charts/pega/templates/_pega-aks-ingress.tpl

```diff
@@ -0,0 +1,24 @@
+{{- define "pega.aks.ingress" -}}
+# Ingress to be used for {{ .name }}
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ .name }}
+  namespace: {{ .root.Release.Namespace }}
+  annotations:
+    # Ingress class used is 'azure/application-gateway'
+    kubernetes.io/ingress.class: azure/application-gateway
+    # Ingress annotations for aks
+    appgw.ingress.kubernetes.io/cookie-based-affinity: "true"
```

Comment:
Is there a cookie timeout we can apply to this ingress?

Reply:
I don't see an API for it, so I submitted an issue against the Azure gateway project.

https://github.com/Azure/application-gateway-kubernetes-ingress/issues/734